climada/util/yearsets.py

```diff
@@ -25,31 +25,32 @@
 def impact_yearset(event_impacts, sampled_years=None, sampling_vect=None):
 
     """PURPOSE:
-      Create an annual_impacts object containing a probabilistic impact for each year 
-      in the sampled_years list (or a list generated with the length of given sampled_years) 
+      Create an annual_impacts object containing a probabilistic impact for each year
+      in the sampled_years list (or a list generated with the length of given sampled_years)
       by sampling events from the existing input event_impacts.
```

Comment:
Maybe this would make sense?
```suggestion
      by sampling events from the existing input event_impacts with a Poisson distribution centered
      around n_events = sum(event_impacts.frequency).
```